src/apb/engine.py

```diff
@@ -746,6 +746,11 @@ def broker_request(broker, service_route, method, **kwargs):
     if not broker.startswith('http'):
         broker = 'https://' + broker
 
+    if kwargs["cert"] is not None:
```

Comment:
What's happening here? Not immediately clear to me.

Reply:
I read this as users can pass the `--verify` flag to use installed certs, using the `cert` flag to specify a cert forces us to use the specified ca for verification. Is that right?

Reply:
That is exactly right. So the following are acceptable:
```
apb list --secure --ca-path <path>
apb list --ca-path <path>
apb list --secure
```